modules/network/vpc/variables.tf

```diff
@@ -148,9 +148,9 @@ variable "additional_subnetworks" {
 }
 
 variable "secondary_ranges" {
```

Comment:
What happens if a customer doesn't update their existing blueprint to adapt to this change? Can we give them an example on how to modify this ?

Reply:
I have updated the description+readme to add an example on how to perform the migration from old to new template

Reply:
Made the changes backward compatible. Introduced new parameter secondary_ranges_list and retained the older one